content/cn/docs/concepts/services-networking/network-policies.md

```diff
@@ -24,7 +24,7 @@ Pod可以通过相关的网络策略进行隔离。一旦命名空间中有网
 
 ## `NetworkPolicy` 资源
 
-通过[api参考](/docs/api-reference/{{< param "version" >}}/#networkpolicy-v1-networking)来了解资源定义。
+通过[api参考](/docs/api-reference/{{< param "version" >}}/#networkpolicy-v1-networking-k8s-io)来了解资源定义。
```

Comment:
pls remove this from your PR. We don't touch other language translations in the main repo.

Reply:
ok done